packages/atomic/src/components/search/atomic-layout/atomic-search-layout.pcss

```
@@ -22,6 +22,11 @@ atomic-layout-section[section='pagination'] {
   grid-area: atomic-section-pagination;
 }
 
+/**
+ * @prop --atomic-layout-max-search-box-input-width: The maximum width that the search box input will take.
+ * @prop --atomic-layout-max-search-box-double-suggestions-width: The maximum width that the search box suggestions will take when displaying a double list.
+ * @prop --atomic-layout-search-box-left-suggestions-size: When displaying a double list, affects size of the left suggestions.
```

```suggestion
 * @prop --atomic-layout-search-box-left-suggestions-size: When displaying a double list, the size of the left suggestions.
```
I find that a big vague though. By size, are we speaking about height, width, or something else?

---

width, maybe I should adapt it